zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/activity/AdHocSubProcessTest.java

```diff
@@ -133,8 +256,19 @@ public void shouldApplyInputMappings() {
             VariableRecordValue::getValue,
             VariableRecordValue::getScopeKey)
         .contains(
-            tuple("x", "1", processInstanceKey),
-            tuple("local_x", "2", adHocSubProcessActivated.getKey()));
+            tuple("activateElements", "[\"A\"]", processInstanceKey),
+            tuple("activateElements", "[\"A\",\"B\"]", adHocSubProcessActivated.getKey()));
+
+    assertThat(
+            RecordingExporter.processInstanceRecords()
+                .withProcessInstanceKey(processInstanceKey)
+                .limit("B", ProcessInstanceIntent.ELEMENT_ACTIVATED))
+        .extracting(r -> r.getValue().getElementId(), Record::getIntent)
+        .containsSequence(
+            tuple(AD_HOC_SUB_PROCESS_ELEMENT_ID, ProcessInstanceIntent.ELEMENT_ACTIVATING),
+            tuple(AD_HOC_SUB_PROCESS_ELEMENT_ID, ProcessInstanceIntent.ELEMENT_ACTIVATED),
+            tuple("A", ProcessInstanceIntent.ACTIVATE_ELEMENT),
```

Comment:
💭 For the future contributors: may be something like short describedAs would be useful to explain what order is expected here.

Reply:
In general, I agree that `describedAs(..)` is useful to provide additional context in a test. 

In this specific case, I don't see the need for it. The expected order is clearly defined by the assertion `containsSequence(..)`.